src/app/frontend/replicasetlist/replicasetcard.html

```diff
@@ -19,11 +19,22 @@
     <div layout="column">
       <div flex layout="row" layout-align="space-between center"
           class="kd-replicaset-card-title-row">
-          <a ng-href="{{::ctrl.getReplicaSetDetailHref()}}" class="kd-replicaset-card-name" flex>
-            <h3 class="md-title kd-replicaset-card-title">
+        <span layout="row" flex>
+          <a ng-href="{{::ctrl.getReplicaSetDetailHref()}}" class="kd-replicaset-card-name">
+            <span class="md-title kd-replicaset-card-title">
               {{::ctrl.replicaSet.name}}
-            </h3>
+            </span>
           </a>
+          <md-icon class="material-icons kd-error kd-replicaset-card-status-icon"
+                   ng-if="::ctrl.hasWarnings()">
+            error
+            <md-tooltip>One or more pods have failed scheduling</md-tooltip>
```

Comment:
Have failed scheduling? Maybe just say that have error events? This is because it is not always failed scheduling, it can be other errors.